components/feedback/stickybutton.js

```diff
@@ -0,0 +1,73 @@
+const StickyButton = `
+    <style>
+      .sticky-button {
+        font-family: Chakra Petch;
+        font-style: normal;
+        font-weight: 600;
+        font-size: 1em;
+        border: 1.5px solid black;
+        border-radius: 10px;
+        padding: .8em .8em;
+        text-decoration: none;
+        color: black;
+        cursor: pointer;
+        background-color: var(--primary-color);
+        position: fixed;
+        top: 50%;
+        right: 0px;
+        -webkit-transform-origin: 100% 100%;
+        transform-origin: 100% 100%;
+        -webkit-transform: rotate(-90deg);
+        transform: rotate(-90deg);
+      }
+      .sticky-button:hover .sticky-button:focus {
```

Comment:
I don't think the hover is working as intended. Also, I wonder if we could have box shadows such as following to keep things consistent.
```css
.sticky-button {
...
    box-sizing: border-box;
    box-shadow: -10px 10px 0px black;
...
}
.sticky-button:active {
    box-shadow: -5px 5px 0px #000000;
    webkit-transform-origin: 105% 100%;
    transform-origin: 105% 100%;
    -webkit-transform: rotate(-90deg);
    transform: rotate(-90deg);
}
.sticky-button:hover {
    background-color: var(--primary-color-dark);
}
```

Reply:
It's ok also for me ! I don't know it's ok for @fcor

Reply:
Looks good to me @gciandro13 only thing would be that hover effect pointed by @mingir2 😃